src/pages/fi-pv-expenses/forms/PaymentVoucherExpensesForm.js

```diff
@@ -0,0 +1,741 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useForm } from 'src/hooks/form'
+import { ControlContext } from 'src/providers/ControlContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { SystemFunction } from 'src/resources/SystemFunction'
+import { DataSets } from 'src/resources/DataSets'
+import { FinancialRepository } from 'src/repositories/FinancialRepository'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { CashBankRepository } from 'src/repositories/CashBankRepository'
+import CustomTextArea from 'src/components/Inputs/CustomTextArea'
+import { useDocumentType } from 'src/hooks/documentReferenceBehaviors'
+import { formatDateFromApi, formatDateToApi } from 'src/lib/date-helper'
+import WorkFlow from 'src/components/Shared/WorkFlow'
+import { useWindow } from 'src/windows'
+import ExpensesCostCenters from 'src/components/Shared/ExpensesCostCenters'
+
+export default function FiPaymentVoucherExpensesForm({ labels, maxAccess: access, recordId, plantId }) {
+  const [subtotalSum, setSubtotalSum] = useState(0);
```

Comment:
those 3 states no need for them , add them as const
example: 
subTotalSum = formik.values.expenses.reduce((sumsubtotal, row) => {
    const curValue = parseFloat(row.subtotal.toString().replace(/,/g, '')) || 0

    return sumsubtotal+ curValue
  }, 0)